src/index.ts

```diff
@@ -18,6 +19,7 @@ import UI, { UserInterface } from './user-interface';
 const debug = createDebug('devcert');
 
 export interface Options {
+  returnCa?: true | 'read',
```

Comment:
I see the need for options to get both the CA path for `NODE_EXTRA_CA_CERTS` and the contents themselves for `https.Agent`, but this API is a bit confusing to me. I'd rather have two properties:

```suggestion
  getCaPath?: boolean,
  getCaBuffer?: boolean,
```

Fewer magic strings! How do you feel about this?

Reply:
👍 makes sense to me